yt_dlp/extractor/redgifs.py

```diff
@@ -231,7 +231,7 @@ class RedGifsUserIE(RedGifsBaseInfoExtractor):
                 'title': 'lamsinka89',
                 'description': 'RedGifs user lamsinka89, ordered by recent',
             },
-            'playlist_count': 30,
+            'playlist_mincount': 31,
```

Comment:
since we are only requesting 1 page, we should test for an exact count of 80
```suggestion
            'playlist_count': 80,
```